src/common/cockpittemplate.c

```diff
@@ -140,3 +141,19 @@ cockpit_template_expand (GBytes *input,
 
   return g_list_reverse (output);
 }
+
+GList *
+cockpit_template_expand (GBytes *input,
+                         CockpitTemplateFunc func,
+                         gpointer user_data)
+{
+  return template_expand (input, func, "@@", "@@", user_data);
+}
+
+GList *
+cockpit_css_template_expand (GBytes *input,
```

Comment:
This should be moved to the caller, and just call cockpit_template_expand (input, "${", "}", func, user_data);